Jenkinsfile

```diff
@@ -25,14 +25,13 @@ pipeline {
           branch 'master'
       }
       environment {
-        TILLER_NAMESPACE = "${env.stagingNamespace}"
         ISTIO_DOMAIN = "${env.stagingDomain}"
         PRODUCT_NAME = "${env.product}"
       }
       steps {
         container('skaffold') {
           unstash 'build'
-          sh "skaffold deploy -a image.json -n ${TILLER_NAMESPACE}"
+          sh "skaffold deploy -a image.json -n ${env.stagingDomain}"
```

Comment:
I don't think this value will map to the namespace name

Reply:
Fixed